MeetU/MeetU/Views/Account/Login.cshtml

```diff
@@ -48,11 +48,18 @@
 @section Scripts {
     @Scripts.Render("~/bundles/jqueryval")
     <script>
-        $(".js-login-jump-icon").click(function () {
+        if ($(".validation-summary-errors").length > 0) {
```

Comment:
Thanks for review. @rong4188 
Am I allowed to change existed element name in other js files?
So that I can solve this by changing:
**'validation-summary-errors'** to **'js-validation-summary-errors'**